inst/help/BayesianProcess.md

```diff
@@ -0,0 +1,136 @@
+- The direction of causal effects is correctly specified, e.g., *X* -> *M* -> *Y* instead of *X* <- *M* <- *Y*
+
+Furthermore, conditional process models in JASP are conceptualized as directed acyclic graphs (DAGs). Therefore the model should not contain feedback loops.
+
+#### Structural Equation Model Assumptions
+
+Conditional process models in JASP are conceptualized and estimated as structural equation models (SEMs) which make the following assumptions:
+
+- Exogeneity: The parameters of the conditional distribution of dependent (endogeneous) variables given the independent (exogeneous) variables should be unrelated to the parameters of the unconditional distribution of the independent variables. This implies that all omitted causes of the dependent variables are unrelated to the independent variables (pseudo-isolation).
+- Local independence: The absence of covariance between (the residual variances of) two dependent variables implies that, given the other variables in the model, the two variables should be independent. This implies that they have no omitted common causes.
+
+#### Data Assumptions
+
+Bayesian conditional process models in JASP require all dependent (endogeneous) variables in the model to follow a multivariate normal distribution. This implies that:
+
+- All univariate distributions of dependent variables should be normal.
+- All bivariate associations between dependent variables are linear.
+- The distribution of residuals does not depend on the independent variables, i.e., they are homoscedastic.
+
+### Input
+---
+#### Assignment Box
+
+- Dependent: The final dependent variable of interest.
+- Continuous Predictors: Continuous or ordinal predictor variables.
+- Categorical Predictors: Categorical independent predictor variables.
+
+#### Models
+
+This section allows users to specify multiple process models through one of two interfaces. New models can be added by clicking the green `+` button. For each model, the following options are available:
+
+- Input type:
+	- Variables: Models can be specified by adding and specifying paths. Paths indicate causality. For each path, there are four options
+		- From: The variable from which the path starts.
+		- To: The variable at which the path ends.
+		- Process Type: The type process influencing the path.
+			- Mediator: The path is mediated by a third variable. Adds an indirect path through the `Process Variable`.
+			- Moderator: The path is moderated by a third variable. Adds a direct path from the `Process Variable` and the interaction between the `Process Variable` and `From` to the `To` variable.
+			- Confounder: The path is confounded by a third variable. Adds a direct path from the `Process Variable` both to `From` and `To`.
+			- Direct: Adds a direct path from `From` to `To`.
+	- Model number: Models can be specified by choosing a `Hayes model number` and adding the required variables to assignment boxes.
```

Comment:
likewise, change 'model number' into 'Hayes configuration'. This is perhaps also a good place to make a clear reference where the user can get more knowledge on the different Hayes models (i.e., the book and its appendix)